src/shared/dbus-client.c

```diff
@@ -16,10 +16,57 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <strings.h>
+#include <sys/param.h>
 #include <unistd.h>
 
+#include "a2dp-codecs.h"
+#include "hfp.h"
 #include "shared/defs.h"
 
+struct ba_codec {
+	const char *name;
+	const char **aliases;
+};
+
+static const char *aliases_aptx[] = {
+	"apt-x", NULL
+};
+static const char *aliases_aptx_ad[] = {
+	"apt-x-ad", NULL
+};
+static const char *aliases_aptx_hd[] = {
+	"apt-x-hd", NULL
+};
+static const char *aliases_aptx_ll[] = {
+	"apt-x-ll", NULL
+};
+static const char *aliases_aptx_tws[] = {
+	"apt-x-tws", NULL
+};
+
+static const struct ba_codec codecs[] = {
+	{ "SBC", NULL },
+	{ "MP3", NULL },
+	{ "AAC", NULL },
```

Comment:
I think that the simple structure for alias:name mapping should be enough. There will be not much aliases, and such an array will be more simple to maintain. Compiler should optimize duplicated strings anyway.

```suggestion
static const struct {
    const char *alias;
    const char *name;
} codecs[] = {
	{ "SBC", "SBC" },
	{ "MP3", "MP3" },
	{ "aptX", "aptX" },
	{ "apt-X", "aptX" },
```